doc/nrf/libraries/modem/sms.rst

```diff
@@ -23,14 +23,22 @@ In addition, AT commands are also used to send SMS messages.
 SMS notifications are received using AT commands, but those are not visible for the users of this module.
 The module automatically acknowledges the SMS messages received on behalf of each listener.
 
+The SMS module reads the SMS service center number using the ``AT+CSCA?`` AT command.
```

Comment:
I've been struggling to select the terminology for the document for the phone number (and service center number is the same). 3GPP talks about destination address and source address instead of recipient and sender phone numbers. I've kind of made a choice to talk about "number" instead of "address" in the documentation to be more understandable for the average user of SMS library who doesn't know the protocol or 3GPP details.

Although I realize word "address" is used in the the code and even in some parts of the public API, I've stick to the "number" in the documentation of the SMS library.

Reply:
ok, I found it a bit confusing mixing terminology of service center number and service center address, as the latter is what I've seen elsewhere in our documentation etc.. So I thought it might be better to align, but it is ok by me to keep it as is.

Reply:
Äh, seems both terms are used in the public API. I'll change this to address and mention that it means the service center phone number so everybody should be able to understand it.